invenio/modules/records/models.py

```diff
@@ -159,6 +159,31 @@ class RecordMetadata(db.Model):
     record = db.relationship(Record, backref='record_json')
 
 
+class RecordMetadataHistory(db.Model):
+
+    """Represents the revisions of a record if any."""
+
+    __tablename__ = 'record_json_history'
+
+    id = db.Column(
+        db.MediumInteger(8, unsigned=True),
+        db.ForeignKey(Record.id),
+        primary_key=True,
+        nullable=False,
+        autoincrement=True
```

Comment:
I think `autoincrement` is `True` by default? http://docs.sqlalchemy.org/en/improve_toc/dialects/mysql.html#auto-increment-behavior

Different question is, do we want to have autoincrement in this column? Or if it should contain a `recid` autoincrementing is not desirable? Just thinking out loud..